parsl/data_provider/files.py

```diff
@@ -65,7 +66,8 @@ def __repr__(self) -> str:
                   f"scheme={self.scheme} " \
                   f"netloc={self.netloc} " \
                   f"path={self.path} " \
-                  f"filename={self.filename}"
+                  f"filename={self.filename}" \
```

Comment:
this needs a following space to separate it from `query`:

<File at 0x7f442e7bf690 url=falcon://localhost/tmp/file.txt?8000 scheme=falcon netloc=localhost path=/tmp/file.txt filename=file.txtquery=8000>